### ISSUE TITLE
cattr_accessor is defined by Rails

### ISSUE THREAD

**Issue by bkon** [imajes/rack-heartbeat#3]
.... Which makes it impossible to use this middleware with other frameworks (e.g. Sinatra app):

```
gems/rack-heartbeat-1.0/lib/rack/heartbeat.rb:6:in `<class:Heartbeat>': undefined method `cattr_accessor' for Rack::Heartbeat:Class (NoMethodError)
```

**Comment by imajes** [comment]
Good catch. I'll fix this today

**Comment by sgerrand** [comment]
@bkon @imajes: :wave: This was fixed by #4.

**Comment by bkon** [comment]
Closing, as issue is fixed in master
